src/app.ts

```diff
@@ -22,6 +24,7 @@ app.engine("html", ejs.renderFile);
 app.get("/", hubRoute);
 app.get("/hh", howdyHackRoute);
 app.get("/workshops", workshopRoute);
+app.get("/th2019", th2019Route);
```

Comment:
Please make an [express router](https://expressjs.com/en/guide/routing.html) that we can use for additional future/past versions.

Something along the lines of: `https://tamuhack.com/th/<YEAR>`, and something similar for HowdyHack.

Thoughts @gracekfan ?